src/pages/Wishlist.jsx

```diff
@@ -0,0 +1,15 @@
+import "./pages-css/wishlist.css";
+import { data } from "../FAKE.DATA";
+import { ProductCard } from "../components";
+export const Wishlist = () => {
+  return (
+    <main className="wishlist-main">
+      <div className="heading">Wishlist</div>
+      <div className="cards-wrapper">
+        {data.map((x) => (
```

Comment:
Can give more relevant name than 'x' and more specific than 'data'

Reply:
Ok will take care of this from subsequent PRs